ExpressAPI/index.js

```diff
@@ -27,3 +27,9 @@ app.get('/competencies', function(req, res) {
         res.send(rows);
     })
 })
+
+app.get('/responsibilities', function(req, res) {
```

Comment:
what is the point of this route? are you not supposed to be getting responsibilities per band or role? a generic get of all responsibilities seems odd